designs/administrator/4.6/guided-tours/index.md

```diff
@@ -0,0 +1,102 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Guided tours
+
+In 4.6, guided tours will be added to the console. This documentation will outline how tours are designed and should behave. The content of the tour will not be detailed here. Content copy can be found [here](https://docs.google.com/document/d/1qEBDNZYaKF4TDogJLYUqX-8xHCmPCKdaHfKLxMlnZQs/edit?usp=sharing), where it will be reviewed and updated as needed.
+
+## Accessing tours
+Tours can be accessed on both the Administrator and Developer perspectives by going to the **Help menu** and selecting **Guided Tours**. Each of the perspectives will also surface links to the tours in other places around the console. For more detail around how these links will be surfaced in the Developer perspective, view the Developer side documentation.
+
+In the Administrator perspective, a badge will appear on the Overview page informing users that guided tours are available. When clicked, the badge will redirect users to the Guided Tours page.
+
+![1](img/admin1.png)
+
+The Guided Tours page will use a catalog-like design with each tour being shown on a tile. Tiles will provide a brief description of the tour, a time estimate for completing the tour, as well as prerequisites for completing the tour, if any.
+
+![2](img/admin2.png)
+
+If a user clicks on a tour tile, the tour side panel will open. The introductory panel will give users more information and allows the user to start the tour using the **Start tour** button.
+
+![3](img/tour1-screen0.png)
+
+## Tour permissions
+For 4.6, tour permissions will be as follows:
+* If a user can install Operators, show the first Serverless tour (Explore Serverless) which walks users through installing the Serverless Operator.
+* If a user cannot install Operators, do not show the first Serverless tour. Instead show an empty state. The empty state can inform users that tours will be added in future.
```

Comment:
Is there a mockup of what the empty state of this Guided Tours page could look like?

Reply:
This is actually documented on the dev side I believe! @beaumorley @bgliwa01 can you confirm? Or will there also always be a tour available for devs in 4.6? (I maybe remember some mention of non-operator dependent tours?) We'll never have an empty state on admin side for 4.6 because the first Serverless tour will always be there. If we don't need empty states in this release I can take this bit out.

Reply:
Yes. there is an Empty State mockup in the Dev documentation which is now just a link above. The Empty State would display if the user when to Tours through the Help and there were none available. There would be no CTA anywhere else in the Dev console if there were none available. We would only expose a CTA (through a card) when there were tours. We are working on some tours for all privileges and installed capabilities (like a Building a Hello World) but it is not confirmed. I think having the Empty State in our back pocket is a good idea and we should make sure we have that implemented. At least once we go dynamic. These tours will change and content will get stale so good to have!